### ISSUE TITLE
Reward of 1Giota, who  help me to recover my iotas

### ISSUE THREAD

**Issue by jogearg** [iotaledger/wallet#633]
Good night,
In December 2nd of 2017 my iota wallet shows a withdraw at 5:36 pm in The México Time it shows the hash of transaction
PH9GOTUOHP99PVKUDCJQLILUEPNGOGICG9EGBJGQXZOHYXAWWV9POEQVFAIAEHRHF9QQYPWHEOOF99999
and i t shows the address that the iota were send
KYHKKELUX9KLGM9HJXSZZDZJTEGN9ACWKTGRAZPYATFOOXRWAKBNDLBPQPHXAUXY9GKGQDMDOXQRHPYPB9SJQSRYXW.
Can you Help me please it was about 4.9 Gi
I need your support im very sad about this situation can you help me please???
my balance now is Zero

This is the transaction Confirmation

This is the address where it arrives

This is the Branch Trunk and Bunddle

Please help me to recover my iotas i give you all the information you need , I Know you can follow the transaction and recover my iotas please help To Iota Foundation help for this great project, i know that my seed were too short but i know its safe to follow the transactions
Regards this is an excelent project sincerely i appreciate your support
If you help me i want to donate to your Project, I promise to donate in Iota a big amount to your project
Please help me to recover its a big effort forme and my brother, trusting and following this great project for more than a year.

**Comment by iamjoin7** [comment]
I would love to help you out but I don't think there's much I can do to help you.

I see you have 23 Transactions in the Wallet. Did you send IOTA from the same address out twice? If you did then your seed was partially exposed. And after every spending, they would have more information and therefore can access and steal your coins. You can read more about this from here on [Iota Stackexchange](https://iota.stackexchange.com/questions/245/what-information-is-leaked-if-i-reuse-an-address).

I have copy pasted the text from there for you incase you do not have an account there.

`
Come-from-Beyond: Fragments of the private key of that reused address are leaked. An attacker could find a bundle hash (via brute-forcing) that can be signed with the leaked fragments. Luckily for the user, the window for the attack is short (only while the spending transaction is not confirmed). But if more iotas come to the reused address the attacker can spend them right away if he completes the brute-forcing by that time.
`

If the devs identified this they may have your coins but there is also a possibility that a malicious person may have gotten your money as well 😞 

Do try contacting the devs more and see if they can help you out!. You can try on [Slack](http://slack.iota.org) as well! Best of luck and hope you can get your 4.9 Gi back!

**Comment by lodotek** [comment]
How come the slack invite link does not work? Please advise...

**Comment by gitdisrupt** [comment]
Did you resolve your problem?

Seems similar to my concerns ....

Is this the address you SENT your IOTA to? KYHKKELUX9KLGM9HJXSZZDZJTEGN9ACWKTGRAZPYATFOOXRWAKBNDLBPQPHXAUXY9GKGQDMDOXQRHPYPB9SJQSRYXW

This is the address where your IOTA was sent .... do you control that address?

**Comment by fhd75** [comment]
Hello, I am in the same situation, but I was stolen 6,4 Giota...
I offer too 1 Giga if anyone can recover that.
This is the transaction:
https://iotasear.ch/bundle/TXIINGODGPDFODCJDCTTHOFQGRET9XDGXKLGQAEZGVZJHXADC9PFCKL9TPPGLLHKDWSIIRFHEDXPPODSA

In my case, I never used the addresses to send. But my seed was weak; 30 Chars from a song
It was probably a brute force attack. I need the slack of devs too, thanks!!!

Jogearg please let me know if you have any news, thanks

**Comment by mathiasrw** [comment]
> 30 Chars from a song It was probably a brute force attack

Good reason to why security by choice is not a good implementation strategy.

**Comment by fhd75** [comment]
I agree. But not only the right lyrycs, 30 characters of that...is it possible that an attacker be able to know I used only 30? I. That case it would be a security hole.

**Comment by iamjoin7** [comment]
@jogearg @lodotek @fhd75 

This is a Slack Invite link that will be valid for 1000 users. 

[Slack Invite Link](https://join.slack.com/t/iotatangle/shared_invite/enQtMjgzNDMwNTc2ODg3LTYwNmFjODNhOThjZDA3OGIzZWIxYjFjYWY4YjAzYjg1NDVlMGVkMWE3N2EyODRiY2UyYmI5OWI4ZDU5MjY0NjQ)

**Comment by jogearg** [comment]
Hello,

this address i never use, if the iota team can help us maybe we can follow the transactions and recover our iotas if you have the same problem we need to join efforts and contact iota developers they can give us the iotas.

Regards!!!


________________________________
De: gitdisrupt <notifications@github.com>
Enviado: miércoles, 6 de diciembre de 2017 03:49 a. m.
Para: iotaledger/wallet
CC: jogearg; Author
Asunto: Re: [iotaledger/wallet] Reward of 1Giota!!!, to the person that help me to recover my 4.9 Gi!!! (#633)


Did you resolve your problem?

Seems similar to my concerns ....

Is this the address you SENT your IOTA to? KYHKKELUX9KLGM9HJXSZZDZJTEGN9ACWKTGRAZPYATFOOXRWAKBNDLBPQPHXAUXY9GKGQDMDOXQRHPYPB9SJQSRYXW

This is the address where your IOTA was sent .... do you control that address?

**Comment by ScotchWiskey** [comment]
You need to use one of these 4 servers and follow this advise. It worked for me and I am so relieved. I had pending transactions then got Connection Refused messages. Now I got logged in and my balance has been fully updated.

Found advise from Korneto02 and it worked for me and lots of others who could not log in. I was so stressed out and now feel like a million $$$. Try this and post if it works for you. I think it likely will.
hey guys it worked for me go to these site https://iotasalad.org/nodes.... use the nodes there and try all of those, go custom the host and change curl implementation to ccurl...it takes a while like 20-30 minutes says logging in, you just have to wait
